features/step_definitions/steps.rb

```diff
@@ -11,13 +11,15 @@
 end
 
 Given(/^its dish price is "([^"]*)" in our database$/) do |dish_price|
-  Dish.create(price: dish_price)
+  dish = Dish.first(:name => 'Carbonara')
+  dish.update(price: dish_price)
 end
 
 Then(/^I should se a message saying "([^"]*)"$/) do |message|
   expect(page).to have_content message
 end
 
 Given(/^its dish desription is "([^"]*)" in our database$/) do |dish_description|
-  Dish.create(description: dish_description)
+  dish = Dish.first(:name => 'Carbonara')
```

Comment:
Much better. Now I want you to update this command to the correct hash syntax ;-)
(and the one about price)
```
dish = Dish.first(name: 'Carbonara')
```

Reply:
Ok! I used the datamapper suggestion. I'll change it! :)